### PR TITLE
Post "Rereading The Mythical Man-Month"

### PR DESCRIPTION
**Pull request by rachelcarmena**
Some of my notes :wink:

### CODE REVIEW

File: _posts/2018-03-31-rereading-the-mythical-man-month.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+- silver bullet
+- information-hiding
+- end-to-end skeleton
+- communication
+- people
+- peopleware
+
+---
+
+One of my best childhood memories is listening to the stories that my maternal grandfather, a very curious man, told me. However, not only we lose a lot of details over time, but also we age and the same story can provide us different information after years. Last summer I reread _Robinson Crusoe_ by _Daniel Defoe_ after more than 20 years and I discovered an absolutely different book.
```
Comment (richardjwild) [inline _posts/2018-03-31-rereading-the-mythical-man-month.md]: not only we lose -> not only do we lose

File: _posts/2018-03-31-rereading-the-mythical-man-month.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+- end-to-end skeleton
+- communication
+- people
+- peopleware
+
+---
+
+One of my best childhood memories is listening to the stories that my maternal grandfather, a very curious man, told me. However, not only we lose a lot of details over time, but also we age and the same story can provide us different information after years. Last summer I reread _Robinson Crusoe_ by _Daniel Defoe_ after more than 20 years and I discovered an absolutely different book. 
+
+Now I've had the same experience with <a href="http://wiki.c2.com/?MythicalManMonth" target="_blank">The Mythical Man-Month</a> by <a href="http://wiki.c2.com/?FredBrooks" target="_blank">Fred Brooks</a>, the father of the IBM System/360. This book was published in **1975** and it was reissued in its 20th Anniversary with four new chapters. On the one hand, it contains problems which still happen nowadays and it's worthwhile reading, because:
```
Comment (richardjwild) [inline _posts/2018-03-31-rereading-the-mythical-man-month.md]: reissued in its 20th Anniversary -> reissued on its 20th Anniversary

File: _posts/2018-03-31-rereading-the-mythical-man-month.md [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+
+This was the unit of effort used in estimating and scheduling. Men and months were interchanged, when confusing effort with progress. It's said that they can be only interchanged when a task can be partitioned among many workers with no communications among them and no sequential constraints.
+
+* **Systems Test**
+
+Testing phase was the most mis-scheduled part of programming, because it was at the end of the development process. It was thought the solution was to increase the system test time in the original schedule. 
+
+* **Gutless Estimating**
+
+When an omelette, promised in two minutes, has not set in two minutes, the customer has two choices: wait or eat it raw (or partially burned, when turning up the heat). Software managers might have the same courage and firmness than chefs, avoiding false scheduling. _We need to develop and publicize productivity figures, bug-incidence figures, estimating rules, and so on. The whole profession can only profit from sharing such data._
```
Comment (richardjwild) [inline _posts/2018-03-31-rereading-the-mythical-man-month.md]: Software managers might have the same courage and firmness than chefs -> Software managers might have the same courage and firmness as do chefs

File: _posts/2018-03-31-rereading-the-mythical-man-month.md [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+    * Changeability for new cases
+    * Invisibility, not inherently embedded in space and we need several graphs to represent the software
+* **Accidents**: incidental complexities that are imposed by the ways that software is built in the world.
+    * Physical limitations of size and speed
+    * Low levels of abstraction in programming languages
+    * Imperfect specification or interpretation of requirements
+    * Imperfect and inefficient communication among developers
+    * ...
+
+Brooks prized technical developments as _potential silver bullets_: high-level languages advances, OOP, Artificial Intelligence, Expert Systems, _automatic_ programming, graphical programming, program verification (not _error-proof programs_, but _meeting specification_), environments, tools and workstations. On the other hand, he valued _buy vs. build_, requirements refinement and rapid prototyping, incremental development (growing, not building, software) and being great designers, because _software construction is a creative process_.
```
Comment (richardjwild) [inline _posts/2018-03-31-rereading-the-mythical-man-month.md]: Prized? Do you mean appraised maybe? I read it recently too (you inspired me), and unless my memory is faulty, Brooks held his original assertion as still true, that there is still no silver bullet.

Comment (rachelcarmena) [reply _posts/2018-03-31-rereading-the-mythical-man-month.md]: Yes, there is still no silver bullet, but he considers there are "potencial" silver bullets.

Comment (rachelcarmena) [reply _posts/2018-03-31-rereading-the-mythical-man-month.md]: I've changed it by "considered" :wink:

Comment (richardjwild) [reply _posts/2018-03-31-rereading-the-mythical-man-month.md]: Okay, so "prized" means "considers extremely valuable," is that your intended meaning?

Comment (rachelcarmena) [reply _posts/2018-03-31-rereading-the-mythical-man-month.md]: Yes! Thanks @richardjwild !!

**Comment by rachelcarmena** [comment]
Thanks @richardjwild for your valuable help!!

**Review by archel — APPROVED** [review]

**Pull request merged**